src/Database/DBA.php

```diff
@@ -210,9 +212,8 @@ public static function databaseName() {
 	 * @throws \Exception
 	 */
 	private static function logIndex($query) {
-		$a = \get_app();
 
-		if (!$a->getConfigVariable('system', 'db_log_index')) {
+		if (!self::$config->get('system', 'db_log_index')) {
```

Comment:
I renamed it to `self::$configCache` to make it more transparent.
It's really only the configuration-cache out of the config-files (class `ConfigCache`). So no, there shouldn't be any invoked database calls. I know we had some infinite loop problems because of it.